train_segmentation.py

```diff
@@ -282,7 +286,7 @@ def train(train_loader,
     :param batch_size: number of samples to process simultaneously
     :param ep_idx: epoch index (for hypertrainer log)
     :param progress_log: progress log file (for hypertrainer log)
-    :param vis_batch_range: range of batches to perform visualization on. see README.md for more info.
+    :param vis_batch_range: range of batches to perform visualization on. see old_README.md for more info.
```

Comment:
This old_README.md doesn't exist yet. Why rename this here? Just waiting to update the README?

Reply:
Did change automatically with pycharm (that why vim is better)